Add commit information when exporting an archive

When archiving the patchqueue, generate a file describing the patchqueue
commit information.  Also include a file in the base repository that
descibes the base repository commit information. This must be regenerated
each time the base commit is changed. To do that, run the following from
the top of the base repo and refresh this patch:

git log -1 --pretty='format:Commit: %H%nCommit date: %cD%n' <base-commit> > .gitarchive-info

diff --git a/.gitarchive-info b/.gitarchive-info
new file mode 100644
index 0000000..f16b4e41
--- /dev/null
+++ b/.gitarchive-info
@@ -0,0 +1,2 @@
+Commit: dd4534d88b93a911983503851ec9441787a4997d
+Commit date: Sun, 26 Feb 2017 11:08:29 +0100
diff --git a/.gitarchive-info-pq b/.gitarchive-info-pq
new file mode 100644
index 0000000..83e5b86
--- /dev/null
+++ b/.gitarchive-info-pq
@@ -0,0 +1,2 @@
+Commit: dd74044fd0e9b41aae34521337e5e6294e4b6f11
+Commit date: Mon, 9 Apr 2018 10:18:01 +0100
